src/index.html

```diff
@@ -9,6 +9,8 @@
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Colored blocks</h1>
+    <div class = "red block"> </div>
```

Comment:
do not add spaces around `=` in html, this is redundant

Reply:
Thank you. I fixed.